src/view.c

```diff
@@ -214,7 +214,22 @@ view_resize_relative(struct view *view, int left, int right, int top, int bottom
 	newgeo.y -= top;
 	newgeo.height += top + bottom;
 	view_move_resize(view, newgeo);
-	view_set_untiled(view);
+	if (view_is_tiled(view))
+		view_set_untiled(view);
+}
+
+void
+view_move_relative(struct view *view, int x, int y)
+{
+	if (view->fullscreen)
+		return;
+	if (view->maximized)
+		view_maximize(view, false, false);
```

Comment:
An inline comment for the 2nd bool argument would be nice: `view_maximize(view, false,  /*store_natural_geometry*/ false)`